### ISSUE TITLE
Cannot create requests [GUI buttons doesn't work]

### ISSUE THREAD

**Issue by cnekmp** [alexandrehtrb/Pororoca#4]
https://user-images.githubusercontent.com/35117097/166866476-ec757910-df23-4a9e-8675-62e7608689e9.mp4

Please see video for details. Distro: Arch Linux.

**Comment by alexandrehtrb** [comment]
Hello @cnekmp !
Thank you for warning about this! I will fix this as soon as possible!

**Comment by cnekmp** [comment]
@alexandrehtrb lol I've missed that (actually I was expecting to expand automatically  when clicked). Closing this issue. Thanks a lot. 

P.S. Just a tip for future update: Allow selection of multiple requests for deletion (in left pane). I know I could delete `PororocaUserData` folder to "reset" application after the mess I've made, but it will be a nice option for users to mass deletion of selected requests/folders/environments. Adding photo just for example what happened :)
![20220505_222017_1046x1061](https://user-images.githubusercontent.com/35117097/166991368-af0e413d-863f-4dc5-93ae-a61852d966e0.png)
